### PR TITLE
Adds README for propagation-stackdriver

### PR DESCRIPTION
**Pull request by joaoandremartins**

### CODE REVIEW

**Comment by joaoandremartins** [comment]
@adriancole let me know if there is anything missing here!

File: propagation-stackdriver/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# propagation-stackdriver
+
+This is a propagation that behaves like `B3Propagation`, but also extracts the tracing context from the 'x-cloud-trace-context' key.
```
Comment (adriancole) [inline propagation-stackdriver/README.md]: only thing I'd suggest is a code block here showing putting this in your build as from the intro sentence it isn't clear which type people should use and where. so.. make that the next thing and sounds good!

Comment (adriancole) [reply propagation-stackdriver/README.md]: sorry I meant to say "builder" ex Tracer.Builder.propagationFactory(what goes here)

**Comment by adriancole** [comment]
great job

**Pull request merged**